src/views/TabView/TabBarBottom.js

```diff
@@ -89,6 +89,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
         </Animated.Text>
       );
     }
+    
+    if (typeof label === 'function') {
+      const st = [styles.label, { color: inactiveTintColor }, labelStyle];
+      const params = {
+        ...scene,
+        defaultStyles: StyleSheet.flatten(st)
```

Comment:
Sorry i don't get it, i want to make possible to leave all tabs with the same inactive styles but for certain tabs change it color when this tab is active. If i remove this i will lose information about `inactiveTintColor`